src/components/WrapEtherBtn.tsx

```diff
@@ -0,0 +1,269 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+import { DEFAULT_MODAL_OPTIONS, ModalBodyWrapper } from 'components/Modal'
+import Modali, { useModali } from 'modali'
+import { InputBox } from 'components/InputBox'
+import { useForm } from 'react-hook-form'
+import { DEFAULT_PRECISION, formatAmountFull } from '@gnosis.pm/dex-js'
+import BN from 'bn.js'
+import { validatePositiveConstructor, validInputPattern } from 'utils'
+import { TooltipWrapper } from 'components/Tooltip'
+import useSafeState from 'hooks/useSafeState'
+
+export const INPUT_ID_AMOUNT = 'wrapAmount'
+
+const ModalWrapper = styled(ModalBodyWrapper)`
+  > form {
+    > div {
+      margin: 3rem 1.5rem;
+
+      ${InputBox} {
+        position: relative;
+        display: flex;
+        flex-flow: row wrap;
+
+        i {
+          position: absolute;
+          right: 1rem;
+          top: 0;
+          bottom: 0;
+          margin: 0;
+          color: #476481;
```

Comment:
can you use `var(--blah-blah)` here please? for dark mode compat etc.

Reply:
Okay, I think I didn't modify any of this CSS. Anyways, is good advice